src/test/java/uk/ac/ebi/ena/webin/cli/WebinCliSubmissionTest.java

```diff
@@ -1,226 +1,236 @@
+    }
+
+    @Test
+    public void testGenomeSubmissionFlatFileAgp() {
+        ManifestBuilder manifest = genomeManifest()
+                .file("FLATFILE", "valid.flatfile.gz")
+                .file("AGP", "valid.agp.gz");
+        WebinCliBuilder.GENOME.build(GENOME_RESOURCE_DIR, manifest).execute();
+    }
+
+    @Test
+    public void testGenomeSubmissionFlatFileAgpWithInfo() {
+        File infoFile = genomeManifest().build();
+        ManifestBuilder manifest = new ManifestBuilder()
+                .file("FLATFILE", "valid.flatfile.gz")
+                .file("AGP", "valid.agp.gz")
+                .field("INFO", infoFile.getAbsolutePath());
+        WebinCliBuilder.GENOME.build(GENOME_RESOURCE_DIR, manifest).execute();
+    }
+
+    @Test
+    public void testGenomeSubmissionFlatFileWithFormatError() {
+        ManifestBuilder manifest = genomeManifest()
+                .file("FLATFILE", "invalid.flatfile.gz");
+        WebinCli cli = assertWebinCliException(WebinCliBuilder.GENOME, GENOME_RESOURCE_DIR, manifest);
+        new ReportTester(cli).textInFileReport("invalid.flatfile.gz",
+                "ERROR: Invalid ID line format [ line: 1]");
+    }
+
+    @Test
+    public void testGenomeSubmissionFastaWithOneSequenceError() {
+        Path inputDir = WebinCliTestUtils.createTempDir().toPath();
+        Path fastaFile = TempFileBuilder.gzip(
+                inputDir, "test.fasta.gz", ">A\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\n");
+        ManifestBuilder manifest = genomeManifest()
+                .file("FASTA", fastaFile);
+        WebinCli cli = assertWebinCliException(WebinCliBuilder.GENOME, inputDir, manifest);
+        new ReportTester(cli).textInSubmissionReport(
+                "Invalid number of sequences : 1, Minimum number of sequences for CONTIG is: 2");
+    }
+
+    @Test
+    public void testGenomeSubmissionFastaWithOneSequencePrimaryMetagenome() {
+        Path inputDir = WebinCliTestUtils.createTempDir().toPath();
+        Path fastaFile = TempFileBuilder.gzip(
+                inputDir,"test.fasta.gz", ">A\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\n");
+        ManifestBuilder manifest = genomeManifest("primary metagenome")
+                .file("FASTA", fastaFile);
+    }
+
+    @Test
+    public void testSequenceSubmissionTab() {
+        ManifestBuilder manifest = sequenceManifest()
+                .file("TAB", "valid/ERT000003-EST.tsv.gz");
+        WebinCliBuilder.SEQUENCE.build(SEQUENCE_RESOURCE_DIR, manifest).execute();
+    }
+
+    @Test
+    public void testSequenceSubmissionTabWithInfo() {
+        File infoFile = sequenceManifest().build();
+        ManifestBuilder manifest = new ManifestBuilder()
+                .file("TAB", "valid/ERT000003-EST.tsv.gz")
+                .field("INFO", infoFile.getAbsolutePath());
+        WebinCliBuilder.SEQUENCE.build(SEQUENCE_RESOURCE_DIR, manifest).execute();
+    }
+
+    @Test
+    public void testSequenceSubmissionTabWithSampleInOrganismField() {
```

Comment:
- Can we please add a test that has an INVALID sampleId e.g. ERS000000 and make sure we get an error back.